### PR TITLE
 M2-6167 Unit tests for some activity items

### PR DESCRIPTION
**Pull request by DimitrySakalouski**
### 📝 Description

Unit tests for some activity items

🔗 [Jira Ticket M2-6167](https://mindlogger.atlassian.net/browse/M2-6167)

Changes include:

- Add unit tests for check-box, audio-player, audio-stimulus-item, date-picker-item, number-selector, simple-text-input, time-picker-item, time-range-item

### CODE REVIEW

**Comment by DimitrySakalouski** [comment]
https://mindlogger.atlassian.net/browse/M2-3996

File: src/shared/ui/survey/tests/AudioStimulusItem.test.tsx [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+      return {
+        stopPlayer: jest.fn(),
+        removePlayBackListener: jest.fn(),
+        startPlayer: jest.fn(),
+        addPlayBackListener: jest.fn(),
+        pausePlayer: jest.fn(),
+        setSubscriptionDuration: jest.fn(),
+      };
+    },
+);
```
Comment (BamMironov) [inline src/shared/ui/survey/tests/AudioStimulusItem.test.tsx:20]: Let's move it to jest.setup.js

**Review by BamMironov — CHANGES_REQUESTED** [review]
Please move mocks for third-party libraries to jest.setup.js. Apart from that, looks good to me

File: src/shared/ui/survey/tests/AudioStimulusItem.test.tsx [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+    playbackCount: 1,
+    isLoading: false,
+  })),
+);
+
+jest.mock('react-i18next', () => ({
+  useTranslation: jest.fn().mockImplementation(() => ({
+    t: jest.fn().mockImplementation((key: string) => key),
+  })),
+}));
```
[the remark this answers is not in the document]
Comment (anq83) [reply src/shared/ui/survey/tests/AudioStimulusItem.test.tsx:42]: good point, done

**Pull request merged**